doc/classes/Image.xml

```diff
@@ -83,6 +84,7 @@
 			<argument index="2" name="lossy_quality" type="float">
 			</argument>
 			<description>
+				Compresses the image to use less memory. Can not access the data if the image is compressed. Returns error if the compression mode chosen is not available. All compression modes can be found in the numeric constants list. Compression mode [code]COMPRESS_ETC2[/code] needs the compression source specified. [code]COMPRESS_ETC[/code] and [code]COMPRESS_ETC2[/code] needs lossy quality specified. This means that some data is lost when the image is compressed.
```

Comment:
> compression mode chosen

chosen compression mode

> All compression modes can be found in the numeric constants list.

See [code]COMPRESS_*[/code] constants.

> Can not access the data if the image is compressed.

What does this mean. This method can't access the data? Or do you mean that once the image is compressed, *you* will no longer be able to access the pixel data directly?

> Compression mode [code]COMPRESS_ETC2[/code] needs the compression source specified. [code]COMPRESS_ETC[/code] and [code]COMPRESS_ETC2[/code] needs lossy quality specified. This means that some data is lost when the image is compressed.

I think this information belongs more appropriately in the constant descriptions below.